common/src/main/java/com/wynntils/wynn/model/quests/QuestInfo.java

```diff
@@ -121,6 +123,33 @@ public boolean isMiniQuest() {
         return isMiniQuest;
     }
 
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+
+        if (o == null || getClass() != o.getClass()) return false;
+
+        QuestInfo questInfo = (QuestInfo) o;
+
+        // Consider it the same quest just based on "constant" aspects of the quest
+        return new EqualsBuilder()
```

Comment:
In every other hashcode/equals, I opted to use the regular java. Either let's make a PR to switch everything up, or just make this standard and not use the builders here too?

Reply:
Fixed.

Let's skip builders.